spec/rack/sinatra_spec.rb

```diff
@@ -26,4 +26,19 @@ def app
     get('/boom')
     expect(backend.reported_exceptions).to include(an_instance_of(CustomError))
   end
+
+  it 'populates request data into rollbar' do
```

Comment:
you can probably just use rack without sinatra. My thinking is - if it's rollbar-specific, it should be in rollbar spec